src/utils/SocketClient.cxx

```diff
@@ -232,7 +232,7 @@ std::unique_ptr<SocketClientParams> SocketClientParams::from_static(
     std::unique_ptr<DefaultSocketClientParams> p(new DefaultSocketClientParams);
     p->staticHost_ = std::move(hostname);
     p->staticPort_ = port;
-    return std::move(p);
+    return p;
```

Comment:
I'm curious why this was changed, any particular reason?

Reply:
I believe this is covered by C++ copy elision. If it weren't covered, the new code will not compile, since copying a unique_ptr is not possible.

Reply:
The reason for the change is that the std::move statement is redundant. According to the language standards, a return is always a move. The compiler was producing a "redundant move in return statement" warning.